client/my-sites/ads/form-earnings.jsx

```diff
@@ -1,62 +1,52 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	notices = require( 'notices' ),
-	classNames = require( 'classnames' ),
-	debug = require( 'debug' )( 'calypso:my-sites:ads-earnings' );
+import React, { Component } from 'react';
+import notices from 'notices';
+import classNames from 'classnames';
```

Comment:
I rarely alphabetize those, because we usually keep the `React` import at the top.